### PR TITLE
:setup action is not managed only once anymore

### PR DESCRIPTION
**Pull request by StraToN**
After this comment from @mjtorn clarifying the difference between :setup and :ready actions, a change has to be done on these actions.

As per this comment, it appears that :setup is not meant to be run once while :ready is meant to be run each time the scene is loaded. Indeed, this can be easily done with a global variable anyway.

~I was unable to find a clear use case for :setup event so this draft PR deletes it so I'd like to discuss this here first.~

Would it be correct to consider :setup and :ready as "steps" of one event happening when the scene is loaded? If this is correct, what is the exact separation of both steps?

----

Also adds various fixes (worker animations missing, etc).

Still missing: documentation update

### CODE REVIEW

File: addons/escoria-core/game/escoria.gd [hunk 1]
```diff
@@ -315,7 +315,7 @@ func _ev_left_click_on_item(obj, event, default_action = false):
 			"arrived"
 		)
 		
-		self.logger.info("Context arrived: %s" % var2str(context))
+		self.logger.info("Context arrived: %s" % context)
```
Comment (dploeger) [inline addons/escoria-core/game/escoria.gd:318]: Ah, does context return a readable representation when called in a string context? Nice.

File: game/rooms/room03/esc/room03_bridge.esc [hunk 2]
```diff
@@ -1,6 +1,13 @@
 # THIS ROOMS FEATURES AN OPEN BRIDGE WITH A ***BROKEN*** BUTTON TO USE IN ORDER TO OPEN
 
-:setup
+
```
Comment (dploeger) [inline game/rooms/room03/esc/room03_bridge.esc:3]: Hm. Isn't this part of the preparation for the scene? Doesn't that make the bridge flip up or down visibly to the player? (Applicable to all other changed esc files as well)

**Review by dploeger — CHANGES_REQUESTED** [review]

File: addons/escoria-core/game/escoria.gd (hunk 1 above)
Comment (StraToN) [reply addons/escoria-core/game/escoria.gd:318]: In fact, this was causing a crash due to an infinite loop, related to `_get_property_list()`. Not sure why. But yes, `Dictionary` is readable directly when printed.

File: game/rooms/room03/esc/room03_bridge.esc (hunk 2 above)
Comment (StraToN) [reply game/rooms/room03/esc/room03_bridge.esc:3]: Indeed this is not correct. Fixing.

**Review by dploeger — APPROVED** [review]

**Comment by StraToN** [comment]
Thanks!

**Pull request merged**